tools/webpack.assets.json

```diff
@@ -3,6 +3,14 @@
     "api": "./static/js/portico/api.js",
     "katex": ["./node_modules/katex/dist/katex.min.js"],
     "landing-page": "./static/js/portico/landing-page.js",
+    "signup": [
+               "./static/js/portico/signup.js",
+               "./node_modules/jquery-validation/dist/jquery.validate.js"
```

Comment:
This is the unminified version of jquery validate, so running it through script-loader will produce the unminified version. I would switch to the minified version in the dist folder.